src/views/Deposit/components/EnterAmount/index.tsx

```diff
@@ -118,7 +95,7 @@ export const EnterAmount: React.FC<IEnterAmount> = ({ farmSelected, setFarmSelec
           <div id={'wallet-balance'}>
             <P2 color={colors.white}>
               {wpoktBalance
-                ? `Wallet balance: ${bnToDec(new BigNumber(wpoktBalance))} wPOKT`
+                ? `Wallet balance: ${new TokenAmount(wpoktBalance, 18, { symbol: 'wPOKT' })}`
```

Comment:
Instead of declaring a new object (which I'm surprised works haha), we can also use the static method `TokenAmount.format()` which has basically the same API.

Reply:
Fixed in 19d679215f7ab2d86f679d5811c7201ac9489b2a